### DISCUSSION TITLE
Sound Issue , lagging

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#4026] [Hubs Cloud Questions] [answered]
Hello , 
I have problem with sound in my rooms when there is a lot of users on it . 
The sound is lagging, Do you have a solution for that  ? 
I have 4 x c5.12xlarge for 70 users in a same room , do I need to increase them  ? 

Best regards and thx for your reply

**Comment by johnshaughnessy** [comment, marked as answer]
Increasing the number or size of your servers will not help this issue. The problem is client-side, not server side. We recommend roughly 25 users in a room, though this number will vary depending on the content of the room, the bandwidth capabilities of each participant, and the power of the devices each participant uses.

The numbers on the page explaining EC2 instance sizes are estimates of the traffic each server can handle. In a typical scenario though, the experience tends to degrade above 25 active users in a room because of the load of the client. Users of mobile phones will hit this client limitation earlier than for example users on gaming PCs. Some Hubs Cloud operators control the entire experience of their users, including the devices that run the client. These estimates allow Hubs Cloud operators to understand the server limits in a best-case scenario regarding client performance.

We can't easily provide similar estimates for client limitations because of the wide variety of devices we support and spaces, media objects, network conditions in use in a given scenario.
